examples/fscloud/README.md

```diff
@@ -4,10 +4,11 @@ This example demonstrates how to use the [fscloud profile](../../profiles/fsclou
 
 This examples is designed to show case some of the key customization options for the module. In addition to the pre-wired CBR rules documented at [fscloud profile](../../profiles/fscloud/), this examples show how to customize the module to:
 1. Open up network traffic flow from ICD mongodb, ICD Postgresql to the Key Protect private endpoints.
-2. Open up network traffic flow from Schematics to Key Protect public endpoints.
+2. Open up network traffic flow from Schematics to Key Protect private endpoints.
 3. Open up network traffic flow from a block of IPs to the Schematics public endpoint.
 4. Open up network traffic flow from the VPC created in this example to ICD postgresql private endpoints.
-5. Customize the rule description for `kms` and the zone name for `codeengine`.
+5. Customize the rule description for `kms` and the zone name for `codeengine` and `cloud-object-storage`.
+6. Added optional location for `codeengine` and `cloud-object-storage` serviceRefs.
```

Comment:
perhaps specify explicitly which locations in this case, and also avoid using the term serviceRefs here as most users won't understand. Explain as if you were talking to a users not familiar with details of CBR - eg: "open up traffic flow from .. to ... only in location ... and leave the flow closed in other location."

Reply:
This is done.